index.js

```diff
@@ -50,7 +50,12 @@ class NavigationBar extends Component {
   }
 
   componentWillReceiveProps(props) {
-    customizeStatusBar(this.props.statusBar);
+    if (props.statusBar
+        && ((props.statusBar.hidden !== this.props.statusBar.hidden)
+            || (props.statusBar.style !== this.props.statusBar.style)
```

Comment:
Is there any benefit over just
```js
const hasChanged = props.statusBar....;
```
We don't have any plans in using this function elsewhere ;)